api/server/sdk/volume_migrate.go

```diff
@@ -184,6 +210,23 @@ func (s *VolumeServer) Cancel(
 	return &api.SdkCloudMigrateCancelResponse{}, nil
 }
 
+func (s *VolumeServer) filterStatusResponseForPermissions(
+	ctx context.Context,
+	resp *api.CloudMigrateStatusResponse) *api.CloudMigrateStatusResponse {
+	// Filter out volumes the user doesn't have access to
+	for _, cluster := range resp.Info {
+		for j, migrateInfo := range cluster.List {
+			if err := checkAccessFromDriverForVolumeId(ctx, s.driver(ctx),
```

Comment:
Just thought - instead of doing an `Inspect` for every Volume in the MigrateInfo, is it instead acceptable to do an `AllVolumes := Enumerate` to get all volumes the user has read access to, and compare `MigrateInfo` against that? Less driver calls. Do you think this would be a better way to check for Access for many volumes? @lpabon

Reply:
@lpabon addressed this feedback.